cli/integration_tests/basic_monorepo/monorepo/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "monorepo",
+  "workspaces": [
+    "apps/**",
+    "packages/**"
```

Comment:
There is no `packages/` , should we omit this?

Reply:
`util` is under the packages directory

Reply:
oh oops i missed that, my bad.